### ISSUE TITLE
Share a voice sample you created with this repository

### ISSUE THREAD

**Issue by ErfolgreichCharismatisch** [Rayhane-mamah/Tacotron-2#191]
Can someone share a voice sample he created with this repository based on a given and/or a custom set of voice files

**Comment by Yeongtae** [comment]
check #183

**Comment by tdplaza** [comment]
I have trained Tacotron with 77,000 steps, wavenet with 165,000 steps 
[wavenet-audio-speech-mel-00001.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2364889/wavenet-audio-speech-mel-00001.zip)
There is a bit noise at the end. Trying to fix this.
I'm working on Korean Language.

**Comment by Yeongtae** [comment]
[wavenet-audio-mel-7.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2364929/wavenet-audio-mel-7.zip)
Here is another example which is Korean language.

**Comment by begeekmyfriend** [comment]
G&L result, in Chinese mandarin.
[wav-100000-linear-16000.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2364954/wav-100000-linear-16000.zip)

**Comment by ErfolgreichCharismatisch** [comment]
Are the sentences in the training sets or are those new sentences?

**Comment by ErfolgreichCharismatisch** [comment]
> I'm working on Korean Language.

Because it doesn't exist on your operating system or your don't like the voice?

**Comment by tdplaza** [comment]
> Because it doesn't exist on your operating system or you don't like the voice?
Not sure about your questtion

**Comment by ErfolgreichCharismatisch** [comment]
> Not sure about your questtion
Do you use Tacotron because korean doesn't exist on your operating system like Microsoft Mark, Hazel, David or Zira or you just don't like the provided voices and want to have your own custom voice speak korean?

**Comment by lkfo415579** [comment]
@begeekmyfriend is this generated from wavenet model? or Tacotron-1 model only? would you give more information about it plz?

**Comment by tdplaza** [comment]
@Yeongtae How long does your model take to synthesize a batch of sentences? My model takes about 30-35 minutes to synthesize a batch of text sentences (about 8 sentences). Is it too slow?

**Comment by ErfolgreichCharismatisch** [comment]
Can some of you answer my question at https://github.com/Rayhane-mamah/Tacotron-2/issues/193 ?

**Comment by osungv** [comment]
Hi :)

My name is ohsung kwon and I'm a M.S. student in Yonsei University

I failed to train the wavenet model for Korean synthesis.

Could you let me know about your successful wavenet training?
I want to know about the 'hparams.py' of your T2 model

Thank you.

Sincerely,
Kwon

**Comment by osungv** [comment]
@Yeongtae 

Hey, I'm working T2 with Korean DB.
If you can share your settings, I want to know about your korean embedding for text encoder and hparams settings.

**Comment by tdplaza** [comment]
This is my hyperparams setting.
[hparams.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2374053/hparams.zip)
@osungv 
I have made few changes, not too much. You should post your problems, so others can help.

**Comment by osungv** [comment]
@tdplaza 

Thank you for your uploading files.

After I do modified experiment with your hparams, and then I'll post the remained or addressed issues in here.

I do my experiment with 4 hour datasets. How's yours? 
Is any reason for my problem less quantity of my datasets?

**Comment by tdplaza** [comment]
I used KSS dataset: https://www.kaggle.com/bryanpark/korean-single-speaker-speech-dataset.
it is about 12 hours of data. Now I'm training Wavenet using only 1 hour of data. 
 and this is the evaluated sound at 55000 time step.
[step-55000-pred.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2374145/step-55000-pred.zip)

**Comment by osungv** [comment]
@tdplaza 

It means that you used 12 hours of data for spectrogram prediction network and 1 hours of them for waveNet, right?

And I also have a question. Did you train both models separately ?

**Comment by tdplaza** [comment]
no. I mean I have trained both Tacotron and Wavenet with 12 hours data. the result is quite good. 

Now I'm trying both of them with less training data.

and Yes, I trained them separately. Because I just need they run fine, before constructing end-to-end system.

**Comment by lkfo415579** [comment]
@tdplaza Thank you for your korea model parameters sharing.  I understand that you are using **8 bit** waveform output, when i tried this setting, the latest committed program will produce gradient explosion. (loss from **2.7 -> 0.0000000...**)  the program will **stop** before the first saving checkpoint. What version of the program that you are using?

**Comment by osungv** [comment]
@tdplaza 

I have a question about 'use_lws'.
Is that an important parameter for training WaveNet?

**Comment by tdplaza** [comment]
@osungv https://ieeexplore.ieee.org/document/7572016/
Weighted sum is about decomposition signal, use in stft (I guess)
I do not have knowledge about sound, but in my experience, using localized weighted sum results better sound.
The number of training steps I have posted above.

@lkfo415579 yes, I synthesize wavenet in batch of 8 input. I use tacotron-2 old version and wavenet newest one.

**Comment by osungv** [comment]
@tdplaza 

Thank you for your kindness. 

I understand that 'lws' makes the sound quality better when generate waveform using linear or mel-spectrogram. The point I said is that the 'lws' affect the wavenet's training and the model's output quality.

How do you think about that?

**Comment by lkfo415579** [comment]
@tdplaza what is your avg.loss? why do i train wavenet model get exploded every time?...

**Comment by Yeongtae** [comment]
> @Yeongtae How long does your model take to synthesize a batch of sentences? My model takes about 30-35 minutes to synthesize a batch of text sentences (about 8 sentences). Is it too slow?

Wavenet synthesis is so slow.
Because wavenet must predict 22100 samples to generate an wav which is 1 sec.
A V100 generate 100~300 sample during 1 sec.

**Comment by tdplaza** [comment]
@lkfo415579 my tensorflow version is 1.9.0. but I dont think that is your problem.
my Tacotron loss is ~0.20 and wavenet loss is ~ 1.5

**Comment by dream-will** [comment]
> G&L result, in Chinese mandarin.
> [wav-100000-linear-16000.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2364954/wav-100000-linear-16000.zip)
May I ask you what's the dataset you used for traing?

**Comment by begeekmyfriend** [comment]
@dream-will 1W clip Chinese mandarin records. Only for trivial test not for commercial use.
```shell
wget https://weixinxcxdb.oss-cn-beijing.aliyuncs.com/gwYinPinKu/BZNSYP.rar
```

**Comment by begeekmyfriend** [comment]
I just use Pinyin. But note the pinyin has no punctuation while in the Chinese mandarin text it has.

**Comment by terryyizhong** [comment]
> ```shell
> ```shell
> wget https://weixinxcxdb.oss-cn-beijing.aliyuncs.com/gwYinPinKu/BZNSYP.rar
> ```
> 
> 
>       
> ```

Hi @begeekmyfriend  how can you get male voice using biaobei? May I ask which branch of your repository can get such voice of wav-100000-linear-16000.zip ? mandarin-griifin or mandarin-new? 
THanks!

**Comment by begeekmyfriend** [comment]
That is my private corpus with copyright.

**Comment by CorentinJ** [comment]
There are some samples in this video of my real-time voice cloning project: https://www.youtube.com/watch?v=-O_hYhToKoA
